cmd/kubeadm/app/phases/token/bootstrap_test.go

```diff
@@ -47,6 +47,22 @@ preferences: {}
 users:
 - name: kubernetes-admin`
 
+const testIPv6Config = `apiVersion: v1
```

Comment:
Just a thought... an alternative to having two, almost identical, strings for the config, would be to use the template package and a slice of IP addresses inside the loop L131-132.